package.json

```diff
@@ -1,62 +1,62 @@
 {
-  "name": "project-oregano",
-  "private": true,
```

Comment:
@CIPHERTron @riteshsp2000  can we have a workspace specific eslint/prettier config for `lf` instead of `crlf`  for consistency

Reply:
sure... ill check